<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: lightgrey;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>571</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightblue;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>CONTROLE</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>541</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: pink</string>
      </property>
      <property name="title">
       <string>Inserir Dados de Animais</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Script</family>
        </font>
       </property>
       <property name="text">
        <string>Animal:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_animal">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Script</family>
        </font>
       </property>
       <property name="text">
        <string>Nome:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Script</family>
        </font>
       </property>
       <property name="text">
        <string>Idade:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Script</family>
        </font>
       </property>
       <property name="text">
        <string>Vacinado:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>60</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Script</family>
        </font>
       </property>
       <property name="text">
        <string>Castrado:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>10</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Script</family>
        </font>
       </property>
       <property name="text">
        <string>SITUAÇÃO</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_nome">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_idade">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbox_vacinado">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>30</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Script</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <item>
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NAO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SIM</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="cbox_castrado">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>60</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Script</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <item>
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NAO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SIM</string>
        </property>
       </item>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>30</y>
         <width>120</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="lbl_situacao">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>521</width>
         <height>30</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_confirm">
          <property name="styleSheet">
           <string notr="true">background-color: LIGHTBLUE;</string>
          </property>
          <property name="text">
           <string>Confirmar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_cancel">
          <property name="styleSheet">
           <string notr="true">background-color: LIGHTBLUE;</string>
          </property>
          <property name="text">
           <string>Cancelar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tbl_Animal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>541</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>89</number>
      </attribute>
      <column>
       <property name="text">
        <string>Animal</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Idade</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Vacinado</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Castrado</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Situação</string>
       </property>
      </column>
     </widget>
     <zorder>tbl_Animal</zorder>
     <zorder>groupBox</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>ESTATÍSTICAS</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_adocao">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>511</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
      <property name="title">
       <string>Animais prontos para adoção:</string>
      </property>
      <widget class="QLabel" name="lbl_pronto_pra_adocao">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>45</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>511</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
      <property name="title">
       <string>Animais vacinados:</string>
      </property>
      <widget class="QLabel" name="lbl_vacinado">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>350</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
      <property name="text">
       <string>CALCULAR ESTATÍSTICAS</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>611</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightgrey;</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>551</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Script</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">COLOR:WHITE</string>
     </property>
     <property name="text">
      <string>Adoção Legal</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>571</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="text">
        <string>Ordenar Por Tipo de Animal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="text">
        <string>Ordenar Por Nome</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>pushButton_2</zorder>
    <zorder>pushButton_3</zorder>
   </widget>
   <zorder>widget</zorder>
   <zorder>tabWidget</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>590</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar_Animais"/>
    <addaction name="actionCarregar_Animais"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalvar_Animais">
   <property name="text">
    <string>Salvar Animais</string>
   </property>
  </action>
  <action name="actionCarregar_Animais">
   <property name="text">
    <string>Carregar Animais</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
